internal/common/structures.go

```diff
@@ -13,6 +13,11 @@ import (
 func ExpandStringInterfaceList(i []interface{}) []*string {
 	vs := make([]*string, 0, len(i))
 	for _, v := range i {
+		if v == nil {
+			vs = append(vs, nil)
```

Comment:
If the value is nil, it seems to be nice to drop it instead of including in the result

Reply:
If drop a value, diff occurs when running `terraform plan` after `terraform apply`

```
ex)
Terraform will perform the following actions:

  # ncloud_nas_volume.test will be updated in-place
  ~ resource "ncloud_nas_volume" "test" {
      ~ server_instance_no_list        = [
          + "",
            "19973305",
        ]
        # (13 unchanged attributes hidden)
    }
```